ui/packages/catalog-ui-search/src/main/webapp/component/lightbox/result/lightbox.result.view.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global require*/
+var LightboxView = require('./../Lightbox.view');
+var $ = require('jquery');
+
+module.exports = LightboxView.extend({
+    className: 'is-result-form',
```

Comment:
❓ Why do we need a whole new lightbox view? I feel like we could just stick with using the old lightbox view and add a class to it like you do [here](https://github.com/codice/ddf/pull/3192/files/063a90f7e52c711cd32a7d3360c94254dba80868..2f4937176c63450368dc05a583ca91714ea96224#diff-5f1ab184f26124726515c4d001db1faaR99).

You could also move that new CSS into the existing `lightbox.less` as well

Reply:
I feel like we should be able to keep the same view, but just attach the className dynamically similar to the example @mackncheesiest.

https://stackoverflow.com/questions/18391607/setting-dynamic-css-style-properties-of-a-backbone-view